### PR TITLE
Jekyll-ize the repo

### PR DESCRIPTION
**Pull request by adborden**
Resolves https://github.com/18F/foia-recommendations/issues/125

I haven't converted all the content to Jekyll, only the index page. I took a look at how the other main pages would be updated, but I think that will land in another PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,23 @@ discovery sprint with the Department of Justice.
 
 ## Development
 
+Install the dependencies.
+
+    $ gem install bundler
+    $ bundle install
```
Comment (bsweger) [inline README.md]: Thanks all of this Jekyll-izing work @adborden! Was running through these instructions and got the following message:
```
adb-jekyll $ bundle install                                          [11:12:04]
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies...
activesupport-5.1.2 requires ruby version >= 2.2.2, which is incompatible with
the current version, ruby 2.2.0p0
```

I'm sure I just need to make some updates (I run ruby stuff mayyybe once a year). Do you think it's worth adding some high-level notes about what technologies and versions we're using + any prerequisites or assumptions we have for the local dev instructions?

For example: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/README.md#getting-started

Comment (adborden) [inline README.md]: Absolutely. I'm hoping 18F has some canned instructions for managing ruby versions and will include that in the docs. FWIW, Ruby 2.3.4 is expected.

File: www.foia.gov/_layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+---
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```
Comment (hbillings) [inline www.foia.gov/_layouts/default.html]: I wonder if we should either call this something like `legacy-layout.html` template, or templatize this further to allow for easy extraction of the existing code. Seems like once we start porting over new design and structure, having this in the default template might be confusing.

File: www.foia.gov/foia-style.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+//TODO the legacy styles should be removed as we move the site to a unified
+// visual design
+@import "foia-legacy.css";
+
+// SMACSS https://smacss.com/
+@import "base";
+@import "layout";
+@import "modules";
+@import "theme";
```
Comment (hbillings) [inline www.foia.gov/foia-style.scss]: 💯 to this approach. I think this sets us up well to bring everything over to this new structure and modify as needed.

File: www.foia.gov/_layouts/default.html (hunk 2 above)
Comment (adborden) [inline www.foia.gov/_layouts/default.html]: :+1: that makes sense, do you mind if we punt until we know more about the new template will contain?

File: www.foia.gov/foia-style.scss (hunk 3 above)
Comment (adborden) [inline www.foia.gov/foia-style.scss]: Great, Acquia is also suggested SMACSS.

**Comment by adborden** [comment]
@bsweger #ruby suggested using tools that I'm not even familiar with and that aren't documented in 18F's guides, so I just added a short note about tools I'm familiar with that exist.

File: www.foia.gov/_layouts/default.html (hunk 2 above)
Comment (hbillings) [inline www.foia.gov/_layouts/default.html]: 👍

**Pull request merged**